physics/ideal_gas_law.py

```diff
@@ -53,6 +53,40 @@ def volume_of_gas_system(moles: float, kelvin: float, pressure: float) -> float:
     return moles * kelvin * UNIVERSAL_GAS_CONSTANT / pressure
 
 
+def temperature_of_gas_system(moles: float, volume: float, pressure: float) -> float:
+    """
+    >>> temperature_of_gas_system(2, 100, 5)
+    30.068090996146232
+    >>> temperature_of_gas_system(11,5009,1000)
```

Comment:
```suggestion
    >>> temperature_of_gas_system(11, 5009, 1000)
```
Formatting nitpick, would be nice but not super important